python/lsst/pipe/base/executionButlerBuilder.py

```diff
@@ -243,4 +249,3 @@ def buildLightweightButler(butler: Butler,
     newButler = _setupNewButler(butler, outputLocation, dirExists)
 
     newButler = _import(yamlBuffer, newButler, inserts, run, butlerModifier)
```

Comment:
Why is this returning a Butler? It doesn't create one.

Reply:
Good catch, this was sort of twisted up in a refactor, I intended to have it return the newly built butler in case someone (perhaps bps) wanted to make use of it. I will fix that up